internal/service/streaminstance/resource_stream_instance.go

```diff
@@ -83,6 +92,16 @@ func (r *streamInstanceRS) Schema(ctx context.Context, req resource.SchemaReques
 				ElementType: types.StringType,
 				Computed:    true,
 			},
+			"stream_config": schema.SingleNestedAttribute{
+				Optional: true,
+				Computed: true,
```

Comment:
Should we mark this attribute with `ForceNew: true` given that it is not supported in the update operation?

Reply:
In the update operation this attribute is not used at all(PATCH body does not include StreamConfig). Would we need to put `ForceNew: true` anyways? Also, doing a quick check, ForceNew does not seem to be a valid attribute for schema.SingleNestedAttribute